cryptography/src/sha256.rs

```diff
@@ -42,6 +49,12 @@ impl Hasher for Sha256 {
     fn len() -> usize {
         DIGEST_LENGTH
     }
+
+    fn random<R: rand::Rng + rand::CryptoRng>(rng: &mut R) -> Digest {
+        let mut digest = [0u8; DIGEST_LENGTH];
+        rng.fill_bytes(&mut digest);
+        digest.to_vec().into()
+    }
```

Comment:
In this repository, the `runtime` is often tasked with providing randomness. We do this so that we can deterministically simulate the code running on it.